airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -3207,25 +3229,24 @@ definitions:
         enum: [RecordSelector]
       extractor:
         anyOf:
-          - "$ref": "#/definitions/CustomRecordExtractor"
           - "$ref": "#/definitions/DpathExtractor"
+          - "$ref": "#/definitions/CustomRecordExtractor"
       record_filter:
         title: Record Filter
         description: Responsible for filtering records to be emitted by the Source.
         anyOf:
-          - "$ref": "#/definitions/CustomRecordFilter"
           - "$ref": "#/definitions/RecordFilter"
+          - "$ref": "#/definitions/CustomRecordFilter"
       schema_normalization:
         title: Schema Normalization
         description: Responsible for normalization according to the schema.
         anyOf:
           - "$ref": "#/definitions/SchemaNormalization"
           - "$ref": "#/definitions/CustomSchemaNormalization"
-        default: None
       transform_before_filtering:
+        title: Transform Before Filtering
         description: If true, transformation will be applied before record filtering.
         type: boolean
-        default: false
```

Comment:
This is a somewhat breaking change, fixed here: https://github.com/airbytehq/airbyte-python-cdk/pull/480/files#diff-1d9bc4ca384e5c00867da05e9db9d919aba908c59bb73aab83883b9ed20def05R2839-R2842